Algorithm.CSharp/CoarseFineOptionUniverseChainRegressionAlgorithm.cs

```diff
@@ -144,28 +144,6 @@ public override void OnEndOfAlgorithm()
         /// </summary>
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "13"},
```

Comment:
This doesn't look right?

Reply:
Whoops yes, something was off here. It has been updated after the latest changes

Reply:
Still missing I think?